domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/CountyCourtJudgment.java

```diff
@@ -41,31 +43,31 @@
     @Valid
     private final StatementOfTruth statementOfTruth;
 
+    private final CountyCourtJudgmentType ccjType;
+
     @Builder
     public CountyCourtJudgment(
         LocalDate defendantDateOfBirth,
         PaymentOption paymentOption,
         BigDecimal paidAmount,
         RepaymentPlan repaymentPlan,
         LocalDate payBySetDate,
-        StatementOfTruth statementOfTruth
+        StatementOfTruth statementOfTruth,
+        CountyCourtJudgmentType ccjType
```

Comment:
What about the existing `CountyCourtJudgment` data in `claim-store` database? Migration scripts?

Reply:
added

Reply:
Great thanks